src/types/props.ts

```diff
@@ -0,0 +1,5 @@
+import { Todo } from './todo';
+
+export interface Props {
+  todos: Todo[];
+}
```

Comment:
You don't need to create files for your props interfaces, keep the along with component